index.html

```diff
@@ -14,239 +14,320 @@
+                                        website. Look through some of my work and experience! if you like what you see
+                                        and have a project you need coded, dont hesitate to contact me.
+                                    </p>
+                                    <h3>LETS CONNECT</h3>
+                                    <ul class="social-media48">
+                                        <li> <a href="#"><img src="./images/github.jpg" alt="github"></a> </li>
+                                        <li> <a href="#"><img src="./images/medium.jpg" alt="Linkedin"></a> </li>
+                                        <li> <a href="#"><img src="./images/twitter.jpg" alt="medium"></a> </li>
+                                        <li> <a href="#"><img src="./images/Vector.jpg" alt="twitter"></a> </li>
+                                    </ul>
+
+                                    <div class="click52">
+                                        <button>Get my resume</button>
+                                    </div>
+                                </div>
+                                <ul class="oye">
+                                    <li class="names22">
+                                        <h4>Languages</h4><img src="./images/chevron-down-solid.svg" alt="">
+                                    </li>
+
+                                    <li class="agbe">
+                                        <img src="./images/Ellipse-1.jpg" alt="work">
+                                        <h5>Javascript</h5>
+                                    </li>
+
+                                    <li class="agbe">
+                                        <img src="./images/Ellipse-12.jpg" alt="work">
+                                        <h5>HTML</h5>
+                                    </li>
+                                    <li class="agbe">
+                                        <img src="./images/Ellipse-9.jpg" alt="work">
+                                        <h5>CSS</h5>
+                                    </li>
+
+                                    <li class="names77">
+                                        <h4>Frameworks</h4><img src="./images/chevron-right-solid.svg" alt="works">
+                                    </li>
+                                    <li class="names77">
+                                        <h4>Skills</h4><img src="./images/chevron-right-solid.svg" alt="works">
+                                    </li>
+                                </ul>
+                            </section>
+                        </section>
+                    </footer>
+                    <footer class="GridCardlst">
+                        <section class="GridWrapperlst">
+                            <section class="form">
+                                
+                                    <h2>Contact me </h2>
+                                    <p>
```

Comment:
- [ ] Contact me form has has some white space on two sides, as well as other sections. This is happening because you set `html body` margin to `0, 24px`:
```
html body {
    min-width: 368px;
    height: 100%;
    font-family: "Poppins", sans-serif;
    margin: 0 24px;
}
```
I recommend you to delete margins from body for desktop view and leave it only for mobile view. 

CONTACT FORM CURRENTLY: 
<img width="964" alt="Screenshot 2022-02-25 at 15 07 52" src="https://user-images.githubusercontent.com/77038610/155705064-eb30a506-4722-407a-95f3-3242fa4f00d7.png">

Reply:
Hi @ShoiraTa as requested by you, I have:
1.  deleted margins from the body for desktop view and left it only for mobile view and,
2. made sure that all buttons look as provided in Figma:

Reply:
@ekenecf, great job 👍🏻